.cypress/utils/constants.js

```diff
@@ -66,6 +66,7 @@ export const setTimeFilter = (setEndTime = false, refresh = true) => {
       .type('{selectall}' + endTime, { force: true });
   }
   if (refresh) cy.get('.euiButton__text').contains('Refresh').click();
+  cy.get('.euiTableRow').should('have.length.greaterThan', 3);//Replaces Wait
```

Comment:
This assertion ensures the page is loaded before moving on to the 'cy.get()' commands in the testing following this function call. The click of refresh applying the filter adds to the rows and waiting for this change to occur ensures cypress won't search to early before it exist. This was put in place to move towards removing all cy.wait() commands and create more reliable and efficient testing. 
Guidance from - https://glebbahmutov.com/blog/check-more-things/
<img width="1359" alt="Screenshot 2023-05-26 at 9 00 34 AM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/e63fb04b-4114-4d2e-a6ca-546a8c702423">